terraform/applications/securitywiki/mysql.tf

```
@@ -0,0 +1,73 @@
+resource "random_password" "securitywiki_db" {
+  length  = 16
+  special = false
+}
+
+resource "aws_db_subnet_group" "securitywiki_db" {
+  name        = "${var.project}-${var.environment}-rds-subnet-group"
+  description = "${var.project}-${var.environment}-rds-subnet-group"
+  subnet_ids  = flatten([data.terraform_remote_state.vpc.outputs.private_subnets])
+}
+
+resource "aws_db_parameter_group" "securitywiki_db" {
+  name   = "${var.project}-${var.environment}-pg"
```

Taking off `pg` as I thought that might mean postgres. Feel free to keep it if I misunderstood.
```suggestion
  name   = "${var.project}-${var.environment}"
```

---

yeah i thought that a few times too, makes sense to remove.